### PR TITLE
Golden Age node setup changes

### PR DESCRIPTION
**Pull request by Juuddi**
Update node guide for Golden Age changes.

### CODE REVIEW

File: guides/client/node.mdx [hunk 1]
```diff
@@ -121,119 +101,145 @@ To run an instance of go-quai, you'll need to install a few dependencies. You ca
...
+<Steps>
+  <Step title="Open the Config File">
+    <Tabs>
+      <Tab title="Linux">
+        Open the `config.toml` file with your favorite text editor.
 
-- [vim](https://www.vim.org/)
-- [nano](https://www.nano-editor.org/)
-- [Visual Studio Code](https://visualstudio.microsoft.com/)
+        ```bash
```
Comment (Djadih) [inline guides/client/node.mdx:188]: These instructions won't work until go-quai is run at least once, generating a default config.toml. Should include in the instructions an initial run of go-quai before changes are made.

We can also create a run command that just generates the config without starting the node.

**Review by Djadih — CHANGES_REQUESTED** [review]

File: guides/client/node.mdx (hunk 1 above)
Comment (Juuddi) [reply guides/client/node.mdx:188]: I would opt for a run command that generates the config. Running the node once, stopping it, configuring it, and then re-running it is a bit odd in terms of user flow. What would that command be?

Comment (Djadih) [reply guides/client/node.mdx:188]: Yeah I agree but it doesn't exist we'd need to create it. I might be able to get it done this week but can't guarantee.

Comment (Juuddi) [reply guides/client/node.mdx:188]: Okay. Will keep this PR in review until you get a chance to make the command. Just keep me updated.

Comment (Juuddi) [reply guides/client/node.mdx:188]: Should be fixed in the most recent commit.

File: guides/client/node.mdx [hunk 2]
```diff
@@ -145,7 +145,18 @@ There are two types of nodes in Quai: **Global** and **Slice** nodes.
...
+After you've successfully installed go-quai, you'll need to configure your node. Go-quai uses a `config.toml` file to configure the node. If this is your first time running go-quai, you'll need to run the following command to populate the config file:
+
+```bash
+./build/bin/go-quai config
+```
+
+<Warning>
+	You only need to run this command once the first time you run go-quai. Once the config file has been created, you do not need to run the
+	configuration command again.
+</Warning>
```
Comment (Djadih) [inline guides/client/node.mdx]: There's no risk or harm to running it multiple times. I would change this from warning to info so people don't think it resets, and try to run this command to clear their config.

File: guides/client/node.mdx [hunk 3]
```diff
@@ -145,7 +145,18 @@ There are two types of nodes in Quai: **Global** and **Slice** nodes.
...
+
+This will create and populate a go-quai specific `config.toml` file in a directory at the root of your machine.
```
Comment (Djadih) [inline guides/client/node.mdx]: Can you add the XDG default directories if they're not specified elsewhere
Linux: ~/.config/go-quai/config.toml
Mac: ~/Library/Application\ Support/go-quai/config.toml

File: guides/client/node.mdx (hunk 2 above)
Comment (Juuddi) [reply guides/client/node.mdx]: Gotcha, this makes sense.

File: guides/client/node.mdx (hunk 3 above)
Comment (Juuddi) [reply guides/client/node.mdx]: Yep, will add them below.

**Review by Djadih — APPROVED** [review]

**Pull request merged**